application.cpp

```diff
@@ -36,22 +40,32 @@ class application_impl {
             ioctx.run();
          });
 
+         // after creating the thread for handling signals, we can block signals in the current thread
          sigset_t blocked_signals;
-         sigemptyset(&blocked_signals);
-         sigaddset(&blocked_signals, SIGINT);
-         sigaddset(&blocked_signals, SIGTERM);
-         sigaddset(&blocked_signals, SIGPIPE);
-         sigaddset(&blocked_signals, SIGHUP);
+         get_target_sigset(&blocked_signals);
          pthread_sigmask(SIG_BLOCK, &blocked_signals, nullptr);
-#endif
+      }
+
+      void get_target_sigset(sigset_t *blocked_signals) {
```

Comment:
similiar to `&`, `*` should go with type

Reply:
Wouldn't it be easier to have automatic code formatting so we don't waste time for these kind of interactions?

Reply:
Also, there are many instances in the code where `&` and `*` go with the variable name not the type. just look at this one file. Is the preference you mention stated somewhere?

```
./programs/eosio-launcher/main.cpp:    ifaddrs *ifap = 0;
./programs/eosio-launcher/main.cpp:      for (ifaddrs *p_if = ifap; p_if != 0; p_if = p_if->ifa_next) {
./programs/eosio-launcher/main.cpp:          sockaddr_in *ifaddr = reinterpret_cast<sockaddr_in *>(p_if->ifa_addr);
./programs/eosio-launcher/main.cpp:   host_def *find_host (const string &name);
./programs/eosio-launcher/main.cpp:   host_def *find_host_by_name_or_address (const string &name);
./programs/eosio-launcher/main.cpp:   host_def *deploy_config_files (tn_node_def &node);
./programs/eosio-launcher/main.cpp:   void prep_remote_config_dir (eosd_def &node, host_def *host);
./programs/eosio-launcher/main.cpp:    host_def *lhost = nullptr;
./programs/eosio-launcher/main.cpp:host_def *
./programs/eosio-launcher/main.cpp:  host_def *host = nullptr;
./programs/eosio-launcher/main.cpp:host_def *
./programs/eosio-launcher/main.cpp:  host_def *host = nullptr;
./programs/eosio-launcher/main.cpp:host_def *
./programs/eosio-launcher/main.cpp:  eosd_def &instance = *node.instance;
./programs/eosio-launcher/main.cpp:  host_def *host = find_host (instance.host);
./programs/eosio-launcher/main.cpp:   eosd_def &instance = *node.instance;
./programs/eosio-launcher/main.cpp:   host_def *host = find_host (instance.host);
./programs/eosio-launcher/main.cpp:  eosd_def &instance = *node.instance;
./programs/eosio-launcher/main.cpp:  eosd_def &instance = *node.instance;
./programs/eosio-launcher/main.cpp:       ndx = skip_ndx(ndx, l * gap);
./programs/eosio-launcher/main.cpp:      tn_node_def *node = &network.nodes[inst.name];
./programs/eosio-launcher/main.cpp:launcher_def::prep_remote_config_dir (eosd_def &node, host_def *host) {
```

It seems that we don't

Reply:
The location of `&` and `*` is one of the few high level formatting rules we diligently try to enforce. Another example by a different reviewer within the last week is https://github.com/AntelopeIO/leap/pull/635#discussion_r1068464114

A small number of violations of this rule that have snuck in over time aren't really a compelling reason to stop being diligent about it when a vast substantial majority of the code does follow the rule.

Reply:
Thanks @spoonincode , I made the change.